base/system/diskpart/diskpart_msg.mc

```diff
@@ -7856,27 +7877,27 @@ Example:
     UNIQUEID DISK ID=baf784e7-6bbd-4cfb-aaac-e86c96e166ee
 .
 Language=Romanian
-    Displays or sets the GUID partition table (GPT) identifier or master boot
-    record (MBR) signature for the disk with focus.
+    Afișează sau setează identificatorul tabelului de partiție GUID (GPT) sau
+    semnătura Registrului principal de inițializare (MBR) pentru discul
+    în cauză.
 
-Syntax:  UNIQUEID DISK [ID={<DWORD> | <GUID>}]  [NOERR]
+Sintaxă:  UNIQUEID DISK [ID={<DWORD> | <GUID>}]  [NOERR]
 
     ID={<DWORD> | <GUID>}
 
-                For MBR disks, you can specify a four-byte (DWORD) value in
-                hexadecimal form for the signature.
+                Pentru discurile MBR, puteți specifica o valoare de patru
+                octeți (DWORD) formă hexazecimală pentru semnătură.
 
-                For GPT disks, specify a GUID for the identifier.
+                Pentru discurile GPT, specificați un GUID pentru identificator.
 
-    NOERR       For scripting only. When an error is encountered, DiskPart
-                continues to process commands as if the error did not occur.
-                Without the NOERR parameter, an error causes DiskPart to exit
-                with an error code.
+    NOERR       Doar pentru automatizare. Când apare o eroare, programul
```

Comment:
"For scripting only" gets translated here as "doar pentru automatizare", but in other instances as "doar pentru scriere de fișiere script". Please take care of consistency.

Reply:
Finally, I decided to keep "doar pentru scriere de fișiere script" everywhere.